gauntlet/packages/gauntlet-solana-contracts/networks/.env.devnet

```diff
@@ -1,2 +1,5 @@
 NODE_URL=https://api.devnet.solana.com
 
+OCR2=GA6oqeZii6WLgkQbR3NKaP1gsBUPeUvMAqakHiQt8mBX
```

Comment:
At least add a comment above these are program IDs, but I'd rather those env vars be more descriptive perhaps with the `PROGRAM_ID_` prefix.